packages/astro/astro-jsx.d.ts

```diff
@@ -1,4 +1,7 @@
 /// <reference lib="dom" />
+
+import { AstroReloadAttribute } from './dist/@types/astro.js';
```

Comment:
I don't think we can import the types this way in this file. You'd need to use `import()` like the below pattern so that the ambient types don't break.

Reply:
Right! But since we go with data-astro- attributes now I'll revert.